### PR TITLE
Chart

### PR DESCRIPTION
**Pull request by daymos**

### CODE REVIEW

**Comment by daymos** [comment]
svg + d3 = so fiddly

Dropped the legenda and went for tooltip.
Styling is temporary

**Comment by g-sam** [comment]
This looks exciting. A couple of things before merging:
- Could you merge my outstanding prs before making another one? otherwise we'll get conflicts.
- Maybe related to the above: in the latest branch at least `dist` is gitignored. Could you remove those files from the commit?
Cheers.

File: src/client/admin.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  // eslint-disable-next-line no-use-before-define
+  // const compose = (...functr) => inVal => functr.reverse().reduce((acc, fn) => fn(acc), inVal);
+
+  const randomColor = () => (
+    `#${Math.floor(Math.random() * 16).toString(16)}${Math.floor(Math.random() * 16).toString(16)}${Math.floor(Math.random() * 16).toString(16)}`
+  );
+
+  const colorLibGen = data => data.map(el => Object.assign(el, { color: randomColor() }));
+
+  let data = [
```
Comment (g-sam) [inline src/client/admin.js:13]: get these from backend (make sql script for dummy data if you need to)

File: public/css/chart.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.hbox {
+  fill: steelblue;
+}
+.toolTip {
```
Comment (g-sam) [inline public/css/chart.css:4]: namespace these classes (admin-chart or something)

**Pull request closed without merging**